packages/teleterm/src/services/tshd/createClient.ts

```diff
@@ -334,19 +334,29 @@ export default function createClient(
       });
     },
 
-    async getRequestableRoles(clusterUri: string) {
-      const req = new api.GetRequestableRolesRequest().setClusterUri(
-        clusterUri
+    async getRequestableRoles(params: types.GetRequestableRolesParams) {
```

Comment:
i'm confused, this doesn't look like a clean backport. i'm pretty sure the original PR i approved did not include these changes

Reply:
this backport includes the changes from https://github.com/gravitational/webapps/pull/1359 that handle the Connect side which was separately approved and merged into my branch